diff --git a/core/infra/main.prd.bicepparam b/core/infra/main.prd.bicepparam
--- a/core/infra/main.prd.bicepparam
+++ b/core/infra/main.prd.bicepparam
@@ -25,8 +25,8 @@ param logMain = {
 param logMain = {
   deploy: true
   name: 'log-shop-main-prd'
   location: 'northeurope'
-  dataRetentionDays: 90
+  dataRetentionDays: 120
   dailyQuotaGb: '1'
   skuName: 'PerGB2018'
   publicNetworkAccessForIngestion: 'Enabled'
